GpsApp/Top/Topology.cpp

```diff
@@ -256,18 +311,40 @@ void runcycles(NATIVE_INT_TYPE cycles) {
     }
 
 }
-
+*/
 void exitTasks(void) {
     rateGroup1Comp.exit();
     rateGroup2Comp.exit();
     rateGroup3Comp.exit();
+    //blockDrv.exit();
```

Comment:
Is there a reason why blockDrv and pingRcvr's exit functions are commented out?

Reply:
The blockDrv was added back. The pingRcvr is not needed for the GpsApp. Would you like me to remove them completely or add them back?

Reply:
You're right, pingRcvr isn't needed. Can you entirely remove it?